File: delete_train.java
function deleteTrain(id) {
    if(confirm('Are you sure you want to delete this train?')) {
        fetch(`delete_train.php?id=${id}`)
            .then(response => response.json())
            .then(data => {
                if(data.success) {
                    alert('Train deleted successfully!');
                    location.reload();
                } else {
                    alert('Error: ' + data.error);
                }
            });
    }
}

File: delete_train.php
<?php
include 'config.php';

header('Content-Type: application/json');

if(isset($_GET['id'])) {
    $id = $conn->real_escape_string($_GET['id']);
    $sql = "DELETE FROM trains WHERE id = $id";
    
    if($conn->query($sql) === TRUE) {
        echo json_encode(["success" => "Train deleted successfully"]);
    } else {
        echo json_encode(["error" => "Error deleting train: " . $conn->error]);
    }
} else {
    echo json_encode(["error" => "No ID provided"]);
}

$conn->close();
?>